### PR TITLE
NOTIF-192 Silence successful health checks logs

### PR DESCRIPTION
**Pull request by bennetelli**
Health checks produce logs lines that are unnecessary so we want to get rid of these kind of logs.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -124,6 +124,10 @@
             <artifactId>json-schema-validator</artifactId>
             <version>2.2.14</version>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-reactive-jsonb</artifactId>
```
Comment (gwenneg) [inline pom.xml:129]: We already depend on `quarkus-resteasy-reactive-jackson` and I know from experience that mixing both is not a good idea.

**Comment by gwenneg** [comment]
I just found that this kind of filtering was recently added to Quarkus and will be available in version `2.0`: https://github.com/quarkusio/quarkus/pull/16891

The new Quarkus configuration key accepts any valid `Pattern` value, so it can be used to filter whatever we want:
`quarkus.http.access-log.exclude-pattern=/health|/liveliness`

Should we merge something today and replace it with the Quarkus feature later (`2.0` will be available in a few weeks) or wait for the Quarkus feature?

cc @theute @josejulio

**Comment by theute** [comment]
I'm fine either way

**Comment by gwenneg** [comment]
I just did a quick local test to check what I wrote. I am still seeing the `access_log` entries:

```
2021-06-09 09:46:37,520 INFO  [access_log] (vert.x-worker-thread-11) 127.0.0.1 - -noauth- 09/Jun/2021:09:46:37 +0200 "GET /health/live HTTP/1.1" 200 412 "-" "PostmanRuntime/7.28.0"
2021-06-09 09:46:42,568 INFO  [access_log] (vert.x-worker-thread-13) 127.0.0.1 - -noauth- 09/Jun/2021:09:46:42 +0200 "GET /health/ready HTTP/1.1" 200 525 "-" "PostmanRuntime/7.28.0"
2021-06-09 09:46:48,497 INFO  [access_log] (vert.x-eventloop-thread-15) 127.0.0.1 - -noauth- 09/Jun/2021:09:46:48 +0200 "GET /metrics HTTP/1.1" 200 40500 "-" "PostmanRuntime/7.28.0"
```

There's no entry in `Access` when `/health/*` or `/metrics` are called though, but it is there for other API calls:

```
2021-06-09 09:49:52,179 INFO  [Access] (vert.x-eventloop-thread-15) {"date":"Wed, 9 Jun 2021 07:49:52 GMT","method":"GET","uri":"/internal/bundles","http_version":"HTTP/1.1","status":200,"content_length":276,"duration":72,"user_agent":"PostmanRuntime/7.28.0","remote":"127.0.0.1"}
2021-06-09 09:49:52,179 INFO  [access_log] (vert.x-eventloop-thread-15) 127.0.0.1 - -noauth- 09/Jun/2021:09:49:52 +0200 "GET /internal/bundles HTTP/1.1" 200 276 "-" "PostmanRuntime/7.28.0"
```

This wasn't ready for a merge.

**Comment by theute** [comment]
Indeed the logs are still showing up in notifications-ci

**Comment by gwenneg** [comment]
I don't think this PR has been deployed on CI yet. I just started the Jenkins job to do it.

**Comment by gwenneg** [comment]
Oh it was done automatically, my bad 😃

**Comment by bennetelli** [comment]
Yes, I know. I am already working on it.

**Pull request merged**